rep-2000.rst

```diff
@@ -721,39 +721,39 @@ Galactic Geochelone (May 2021 - May 2022)
+" [d] " Distribution-specific (Debian, RPM, etc.) packages will be provided for this platform for packages submitted to the rosdistro.
 
 " [a] " Binary releases are provided as a single archive per platform containing all packages in the Galactic ROS 2 repos file [10]_.
 
 " [s] " Compilation from source.
 
 Middleware Implementation Support:
 
-+--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
-| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                  |
-+==========================+=====================+===============+=============================+================================+
-| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures              |
-+--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
-| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures              |
-+--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
-| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64 |
-|                          |                     |               | and OpenEmbedded            |                                |
-+--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
-| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures              |
-+--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
++--------------------------+---------------------+---------------+-----------------------------+-------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures     |
++==========================+=====================+===============+=============================+===================+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures |
++--------------------------+---------------------+---------------+-----------------------------+-------------------+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures |
++--------------------------+---------------------+---------------+-----------------------------+-------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except RHEL,  | All Architectures |
+|                          |                     |               | Debian, and OpenEmbedded    | except arm64      |
```

Comment:
This will have to be deconflicted with some of the other PRs around this.

Separately, though, I'm wondering if we should change the wording of:
```
All Platforms except RHEL, Debian, and OpenEmbedded
```

to:

```
Ubuntu, Windows, and macOS
```

Reply:
Done in 9eb922f